polishcalctypes.h

```diff
@@ -0,0 +1,16 @@
+#ifndef POLISHCALCTYPES
+#define POLISHCALCTYPES
+
+#include <functional>
+
+namespace PolishCalcComponent {
+
+using value_t = double;
+
+class IOperandsMemory;
+using operation_t = std::function<void(PolishCalcComponent::IOperandsMemory*)>;
```

Comment:
operation_t naming (OperationType) is a bit confusing.
I would rename it just to an `Operation`

Reply:
as default, for typedef and using i use *_t